cluster/openstack-heat/util.sh

```diff
@@ -108,12 +108,20 @@ function create-stack() {
 function upload-resources() {
   swift post kubernetes --read-acl '.r:*,.rlistings'
 
-  echo "[INFO] Upload ${KUBERNETES_RELEASE_TAR}"
-  swift upload kubernetes ${ROOT}/../../_output/release-tars/${KUBERNETES_RELEASE_TAR} \
+  locations=(
+    "${ROOT}/../../_output/release-tars/${KUBERNETES_RELEASE_TAR}"
```

Comment:
Agree, but there's precedent in other places here and elsewhere.

I'd be willing to swap something else in if shown a better pattern.